pkg/capacity/manager.go

```diff
@@ -21,7 +21,8 @@ type (
 
 func hasNotReachedMaxNumberOfSpacesThreshold(counts counter.Counts) spaceProvisionerConfigPredicate {
 	return func(spc *toolchainv1alpha1.SpaceProvisionerConfig) bool {
-		numberOfSpaces := uint(counts.SpacesPerClusterCounts[spc.Spec.ToolchainCluster])
+		//the value of this is not going beyond 100 and it won't overflow, hence its okay to ignore the overflow linter error
+		numberOfSpaces := uint(counts.SpacesPerClusterCounts[spc.Spec.ToolchainCluster]) // nolint:gosec
```

Comment:
the same

Reply:
, I have changed it , but  i dont fully understand this, we are not expecting this to be negative?

Reply:
Okay thanks for explaining